freqtrade/exchange/__init__.py

```diff
@@ -338,6 +338,7 @@ def get_ticker(self, pair: str, refresh: Optional[bool] = True) -> dict:
         # COMMENTED CODE IS FOR DISCUSSION: where should we close the loop on async ?
         # loop = asyncio.new_event_loop()
         # asyncio.set_event_loop(loop)
+        await self._api_async.load_markets()
```

Comment:
i don't think this needs to be done each time - but only once (when we init the exchange object).

for the sync, it's done in `validate_pairs` ... maybe we move this there too?
(and yes, that'll need a `run_until_complete` ...)

Reply:
I don't understand entirely the logic here. all I know is that adding one more API call ahead of fetching candles on every interval doesn't cost that much and makes CCXT happy. also it avoids some debug headaches.

Reply:
well - ccxt.load_markets() loads information about available markets.

it's needed once (and maybe every few hours - but definitely not once per interval).
it would only ever change if new coins are introduced / old ones disappear / ... - but not during "normal" operations.

Reply:
You are right. I just don't see an easy way to (over?)optimize it but propositions are welcome.